lib/index.d.ts

```diff
@@ -354,7 +354,7 @@ export class RateLimiterMemcache extends RateLimiterStoreAbstract { }
 export class RateLimiterUnion {
     constructor(...limiters: RateLimiterAbstract[]);
 
-    consume(key: string | number, points?: number): Promise<RateLimiterRes[]>;
+    consume(key: string | number, points?: number): Promise<Record<string, RateLimiterRes>>;
```

Comment:
@Omers-Frontegg Hi, should it be Array of Record?

Reply:
What do you mean by should?
The docs give an example with a return value of record 🙂

Reply:
@Omers-Frontegg Yes, that's right. Sorry.